accelerator/hpu_accelerator.py

```diff
@@ -0,0 +1,286 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import os
+import pkgutil
+import importlib
+import torch
+
+from .abstract_accelerator import DeepSpeedAccelerator
+
+
+class HPU_Accelerator(DeepSpeedAccelerator):
+
+    def __init__(self):
+        self._name = 'hpu'
+        self._communication_backend_name = 'hccl'
+        try:
+            import habana_frameworks.torch.hpu as hpu
+            hpu.setDeterministic(True)
+            self.hpu = hpu
+        except ImportError as e:
+            print(f"failed importing habana_frameworks.torch.hpu with ImportError: {e=}")
```

Comment:
It might be ideal to use `deepspeed.utils.logger` here instead of `print`?

Reply:
@ShadenSmith thanks, that's a good point. Changed to ValueError which is more appropriate (see the latest commit).